.gitignore

```diff
@@ -69,4 +69,7 @@ source-patch-1*
 evergreen 
 !cmd/evergreen
 
-package-lock.json
\ No newline at end of file
+package-lock.json
+
+# Test config file with secrets
+testdata/test_config.yml
```

Comment:
There's no need to gitignore this. If this is where you put your config file, you should actually move that outside of evergreen's clone directory

Reply:
it has github tokens in it, it can't be pushed

Reply:
What John is saying is that it shouldn't be in the directory at all.

Reply:
removed it